BeeSwift/Gallery/GalleryViewController.swift

```diff
@@ -55,7 +137,19 @@ class GalleryViewController: UIViewController, UICollectionViewDelegateFlowLayou
         self.versionManager = versionManager
         self.goalManager = goalManager
         self.healthStoreManager = healthStoreManager
+        
+        let fetchRequest: NSFetchRequest<Goal> = NSFetchRequest<Goal>(entityName: Goal.entity().name!)
+        fetchRequest.sortDescriptors = [NSSortDescriptor(keyPath: \Goal.urgencyKey, ascending: true)]
```

Comment:
use the preferred sort from the get go


```suggestion
        let fetchRequest: NSFetchRequest<Goal> = NSFetchRequest<Goal>(entityName: Goal.entity().name!)
        fetchRequest.sortDescriptors = preferredSort
```